### DISCUSSION TITLE
Termux: Oh My Zsh! plugin

### DISCUSSION THREAD

**Discussion by b9Joker108** [tmuxinator/tmuxinator#910] [Q&A] [answered]
Hello

I just activated the tmuxinator plugin, in Oh My Zsh! in Termux. I got the unsupported version spiel, as a shell output, but read in the Issues, that this is no concern. I created a 'hummingbird' project:

```zsh
❯ tmuxinator new hummingbird
❯ txs
    WARNING: You are running tmuxinator with an unsupported version of tmux.
    Please consider using a supported version:
    (3.3a, 3.3, 3.2a, 3.2, 3.1c, 3.1b, 3.1a, 3.1, 3.0a, 3.0, 2.9a, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5)

Press ENTER to continue.
Project file at ./.tmuxinator.yml doesn't exist.
```

The following was created: /data/data/com.termux/files/home/.config/tmuxinator/hummingbird.yml
Do I just manually create:  `./.tmuxinator.yml`?

Any assistance greatly appreciated.

Thanking you in anticipation 
Beauford

**Comment by b9Joker108** [comment]
I just just created a global and a local project, and with the two yml configs, tmuxinator started.

**Reply by akofink** [reply, marked as answer]
`tmuxinator start` can be run without arguments to use the local `./.tmuxinator.yml`. Otherwise, specify the project by name:

```
txn hummingbird
txs hummingbird
```
